### ISSUE TITLE
Content vs Page/Attachment/Comment/Blog

### ISSUE THREAD

**Issue by lipkau** [brianbunke/ConfluencePS#46]
According to the API, `Page`, `Attachment`, `Comment` and `Blog` are "`content`".
It would be possible to reflect that in the module:
```
$result = Get-WikiContent -PageID 1234
$result.type -eq "page"
```

I like the dedicated functions and object types for different content types better.
But it is only fare to open it for discussion.

If we would do that, that would mean major breaking changes.
Further: it would not only be hard, but pointless of having custom objects

**Comment by brianbunke** [comment]
Fair point. I far prefer `Get-WikiPage` over `Get-WikiContent -Page`. Even if we're duplicating some code, I think it provides a better end user experience.

Not that I need it, but someone could probably talk me into also providing `Get-WikiContent` as a kind of "superset" function, if a decent use case was presented.

**Comment by lipkau** [comment]
I only see it as an advantage for situations where you have an id and don't know what type it's of.
